.github/workflows/srtool-build.yml

```diff
@@ -0,0 +1,87 @@
+name: Srtool build
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - 'refs/tags/[0-9]+.[0-9]+.[0-9]+*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  SUBWASM_VERSION: v0.21.3
+
+jobs:
+  build-wasm:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        runtime: [peregrine, spiritnet]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Srtool build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.9.2
+        env:
+          PARACHAIN_PALLET_ID: "0x50"
+          AUTHORIZE_UPGRADE_PREFIX: "0x02"
+          AUTHORIZE_UPGRADE_CHECK_VERSION: "true"
+        with:
+          chain: ${{ matrix.runtime }}
+          runtime_dir: runtimes/${{ matrix.runtime }}
+
+      - name: Summary
+        run: |
+          echo '${{ steps.srtool_build.outputs.json }}' | jq > ${{ matrix.runtime }}-srtool-digest.json
+          cat ${{ matrix.runtime }}-srtool-digest.json
+          echo "Compact Runtime: ${{ steps.srtool_build.outputs.wasm }}"
```

Comment:
We can scrape the compact runtime from everywhere, also to avoid potential confusions. We only need the compact-compressed. Also from below.

Reply:
Done